src/Service/CartLinesService.php

```diff
@@ -68,23 +73,29 @@ public function __construct(Mollie $module)
      * @param Cart $cart
      * @return array
      */
-    public function getCartLines($amount, $paymentFee, Cart $cart)
+    public function getCartLines(
+        $amount,
+        $paymentFee,
+        $currencyIsoCode,
+        $cartSummary,
+        $shippingCost,
+        $cartItems,
+        $psGiftWrapping,
+        $selectedVoucherCategory
+    )
     {
-        $oCurrency = new Currency($cart->id_currency);
```

Comment:
Usually it's a good idea to use descriptive variable names, in this case $cartCurrency or something similar.